virttest/libvirt_vm.py

```diff
@@ -2887,10 +2812,10 @@ def prepare_guest_agent(self, prepare_xml=True, channel=True, start=True):
         if not self.is_alive():
             self.start()
 
-        self.install_package('pm-utils', ignore_status=True)
-        self.install_package('qemu-guest-agent')
-
         session = self.wait_for_login()
+        packages = ['pm-utils', 'qemu-guest-agent']
```

Comment:
we can keep `vm.install_package()` method as is and just use `utils_package.package_install(packages, session) ` inside that method, will help us in backward compatibility aswell and we can avoid taking session in all usage places...

Reply:
@sathnaga I agree and it is the better way. Thanks you!